apps/server/src/modules/learnroom/service/dashboard.service.spec.ts

```diff
@@ -0,0 +1,118 @@
+import { createMock, DeepMocked } from '@golevelup/ts-jest';
+import { Test, TestingModule } from '@nestjs/testing';
+import { DashboardElementRepo, IDashboardRepo, UserRepo } from '@shared/repo';
+import { setupEntities, userFactory } from '@shared/testing';
+import { EntityId, LearnroomMetadata, LearnroomTypes } from '@shared/domain/types';
```

Comment:
`EntityId` seems to be not used.
```suggestion
import { LearnroomMetadata, LearnroomTypes } from '@shared/domain/types';
```